<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>801</width>
      <height>541</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>number of wheels</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="wheels">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>40</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>number of doors</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="doors">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>90</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>150</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>motor</string>
      </property>
     </widget>
     <widget class="QComboBox" name="motor">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>150</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>yes</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>no</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="size">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>200</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Small</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Medium</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Large</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>200</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>size</string>
      </property>
     </widget>
     <widget class="QListView" name="listView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>300</y>
        <width>256</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="addWheels">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>32</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>button_icons/add.png</normaloff>button_icons/add.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="addDoors">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>80</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>button_icons/add.png</normaloff>button_icons/add.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="addMotor">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>140</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>button_icons/add.png</normaloff>button_icons/add.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="addSize">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>190</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>button_icons/add.png</normaloff>button_icons/add.png</iconset>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
